sound/soc/sof/intel/hda.c

```diff
@@ -44,6 +44,37 @@
 #define EXCEPT_MAX_HDR_SIZE	0x400
 #define HDA_EXT_ROM_STATUS_SIZE 8
 
+static u32 hda_get_interface_mask(struct snd_sof_dev *sdev)
```

Comment:
typo in commit message: Not all interfaces "are" available...no point "in executing"....

Reply:
OK